oeps/oep-0027-course-metadata-authoring.rst

```diff
@@ -0,0 +1,151 @@
+===================================
+OEP-0027: Course Metadata Authoring
+===================================
+
++-----------------+----------------------------------------------------------------+
+| OEP             | :doc:`OEP-0027 </oeps/oep-0027-arch-course-metadata-authoring>`|
++-----------------+----------------------------------------------------------------+
+| Title           | Course Metadata Authoring                                      |
++-----------------+----------------------------------------------------------------+
+| Last Modified   | 2018-07-13                                                     |
++-----------------+----------------------------------------------------------------+
+| Authors         | Farhanah Sheets, Michael Roytman, Nimisha Asthagiri            |
++-----------------+----------------------------------------------------------------+
+| Arbiter         | Wes Mason                                                      |
++-----------------+----------------------------------------------------------------+
+| Status          | Under Review                                                   |
++-----------------+----------------------------------------------------------------+
+| Type            | Architecture                                                   |
++-----------------+----------------------------------------------------------------+
+| Created         | 2018-07-13                                                     |
++-----------------+----------------------------------------------------------------+
+| `Review Period` | 2018-08-01 - 2018-08-15                                        |
++-----------------+----------------------------------------------------------------+
+
+Context
+-------
+
+An understanding of `edX subdomains`_ is a prerequisite for this OEP.
+
+In the Open edX system, course content authoring occurs in the Studio tool, which focuses mainly on authoring of
+course structure and content within a **Course Run**. A Course Run is discoverable on the edX marketing site via
+the context of a **Catalog Course**. The **Catalog Course** data authoring needs a good user experience.
+
+Today, Catalog Course metadata authoring occurs in multiple places, including the Studio, Ecommerce, and
+Marketing sites. We introduced a new tool, called Publisher, to ease the Catalog Course metadata 
+authoring process. The Publisher tool includes an inflexible workflow engine for the process of provisioning
+a course. Currently, this course publish cycle can take up to fifteen days in order to provision 
+a new course. 
+
+We also run into data synchronization issues between the separate services since there are multiple sources of truth and
+multiple writers of the relevant metadata.
+
+This project will leverage edX's `decoupled frontend architecture`_ plan. In particular, we will be using phase 3 of the plan to separate
```

Comment:
So far, you have stated problems, but have not mentioned what the project is yet.